dist/images/ovnkube.sh

```diff
@@ -698,7 +736,6 @@ ovn-controller () {
   wait_for_event process_ready ovn-controller
   echo "=============== ovn-controller ========== running"
 
-  sleep 4
```

Comment:
why was this there before and why is it ok to remove now?

Reply:
Early code (3.11) needed this to come up. Later added "wait_for_event process_ready ovn-controller" forgot to remove sleep